bitnami/redis/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 15.3.2
+version: 15.3.3
```

Comment:
I think that this is a minor change

Reply:
Fixed: https://github.com/bitnami/charts/pull/7461/commits/2835a1e15271f260ae2754a3f02e222ed4f86981